aws-cpp-sdk-core/include/aws/core/client/ClientConfiguration.h

```diff
@@ -122,6 +122,34 @@ namespace Aws
             */
             Aws::String proxyPassword;
             /**
+            * Username to use for TLS Authentication with a proxy
+            */
+            Aws::String proxyTlsUsername;
```

Comment:
I know we are far from consistent with naming to begin with, but at least we _try_ to stay consistent in the same class/file. So can you s/Tls/TLS and s/Ssl/SSL?